### PR TITLE
CMake: Fix detection of hidapi_VERSION

### PR DESCRIPTION
**Pull request by uklotzde**
https://github.com/mixxxdj/mixxx/pull/3720#issuecomment-802974452

For 2.3 to prevent further merge conflicts.

### CODE REVIEW

File: cmake/modules/Findhidapi.cmake [hunk 1]
```diff
@@ -70,6 +70,21 @@ find_package_handle_standard_args(
   hidapi_INCLUDE_DIR
 )
 
+# Version detection
+if (EXISTS "${hidapi_INCLUDE_DIR}/hidapi.h")
```
Comment (Holzhaus) [inline cmake/modules/Findhidapi.cmake:74]:
```suggestion
if (NOT hidapi_VERSION AND EXISTS "${hidapi_INCLUDE_DIR}/hidapi.h")
```
?

**Comment by Holzhaus** [comment]
CI passes but did you check if this fixes the problem on `main`?

**Comment by uklotzde** [comment]
I did. Cherry picked from my main-based dev branch.

File: cmake/modules/Findhidapi.cmake (hunk 1 above)
Comment (Holzhaus) [reply cmake/modules/Findhidapi.cmake:74]: Ah, forgot about this. @uklotzde do you want to make a new PR?

**Pull request merged**